chart/templates/daemonset.yaml

```
@@ -92,5 +92,13 @@ spec:
             {{- end }}
           resources:
             {{- toYaml $.Values.resources | nindent 12 }}
+          {{- with .Values.extraVolumeMounts }}
+          extraVolumeMounts:
```

```suggestion
          volumeMounts:
```

Same in chart/templates/deployment.yaml

[Pod API Spec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes-1)

---

Thanks!